chrome/content/zotero/xpcom/server/server_localAPI.js

```diff
@@ -983,6 +983,9 @@ async function citeprocToHTML(itemOrItems, searchParams, asCitationList) {
  * @returns {Promise<String>}
  */
 function exportItems(items, translatorID) {
+	if (!Array.isArray(items)) {
+   	 items = [items];
+ 	 }
```

Comment:
Looks like a mix of tabs and spaces here, but this should probably get fixed where `exportItems` is actually being called (presumably [here](https://github.com/zotero/zotero/blob/fefad8a9e34a8f5e7d76fc125fe3175a8ab8d5ab/chrome/content/zotero/xpcom/server/server_localAPI.js#L365)) or where the data is initially set to a single object instead of an array. I'll let @AbeJellinek comment on that.